lrauv_ignition_plugins/src/ScienceSensorsSystem.cc

```diff
@@ -515,11 +682,25 @@ void ScienceSensorsSystem::Configure(
   this->dataPtr->cloudPub = this->dataPtr->node.Advertise<
       ignition::msgs::PointCloudPacked>("/science_data");
 
-  this->dataPtr->node.Advertise("/science_data",
+  this->dataPtr->node.Advertise("/science_data_srv",
```

Comment:
I think I just need to change it back. I used a different name because while trying to understand the code the first time some weeks ago, I was confused because they're the same name, but changing a string didn't change the one I wanted. Maybe it's better to declare a variable and comment that it's used for both the topic and the service.

Reply:
Variable added in c287169